test/C/src/concurrent/WatchdogAction.lf

```diff
@@ -0,0 +1,74 @@
+/**
+ * Test watchdog. This test starts a watchdog timer of 1500ms every 1s. Half the time, it then
+ * sleeps after starting the watchdog so that the watchdog expires. There should be a total of two
+ * watchdog expirations. This version uses an action instead of a reaction to the watchdog.
+ * @author Benjamin Asch
+ * @author Edward A. Lee
+ */
+target C {
+  timeout: 11000 ms
+}
+
+reactor Watcher(timeout: time = 1500 ms) {
+  // Offset ameliorates startup time.
```

Comment:
What startup time is this referring to?

Reply:
The time it takes the program to start up.

Reply:
I think "ameliorates" is a bit odd in this sentence. Did you mean "amortizes"?

Reply:
Replaced with 
```
  // Offset may reduce the likelihood of flakiness if long startup times occur.
```